.envs/environment.yml

```diff
@@ -0,0 +1,15 @@
+name: klaam
+channels:
+  - defaults
+  - conda-forge
+  - pytorch
+  - huggingface
+dependencies:
+  - python=3.8
+  - poetry
+  # klaam dependencies
+  - librosa
```

Comment:
I am really confused which should be considered as a dev_environment vs environmlent and the choices of packages for each.

Reply:
To give more context, `.env/dev_environment.yml` contains the same packages as `.env/environment.yml` but CPU version so we can run CI on GitHub servers.